CMakeLists.txt

```diff
@@ -182,10 +207,15 @@ add_executable(tests
   test/util/graph_verifier_test.cpp
   graph.cpp
   supernode.cpp
+  GraphWorker.cpp
```

Comment:
nit: can you change the name to match our other naming standard? lowercase_with_underscores

Reply:
Sure